.vscode/launch.json

```diff
@@ -177,21 +177,5 @@
         "NODE_ENV": "development"
       }
     },
-
-    {
-      "type": "node",
-      "request": "launch",
-      "name": "Debug TSD",
-      "program": "${workspaceFolder}/node_modules/jest/bin/jest",
-      "args": [
-        "--runInBand",
-        "--no-cache",
-        "--collectCoverage=false",
-        "--config=jest.tsd.js",
-        "--testTimeout=120000"
-      ],
```

Comment:
This config does nothing anymore. It can be removed as well.

Is it worth to have launch configuration for type tests? I mean, in this case there is no run time. Type testing only performs static analysis (similar to `tsc` or `eslint`), so there no way to have breakpoints or to do any data inspection.